src/urllib3/response.py

```diff
@@ -731,7 +737,11 @@ def stream(
 
     @classmethod
     def from_httplib(
```

Comment:
We should be able to remove `from_httplib` in this PR.

Reply:
Really? It seems like a handy class method to have. I would have to take all the logic from `from_httplib` and stick it someone else like `getresponse`? Maybe thats desirable for future plans, but it seems like it would also hurt the ability to have a custom reponse class.

Reply:
Done